### PR TITLE
Add Creators Feature Page

### PR DESCRIPTION
**Pull request by kaitoo1**
### Summary of Changes
Adds the Creators announcement Feature Page.
This is built off of the Posts Feature page and powered by the same CMS mechanic. 
The PR makes the Posts feature page generic so it can be reused for any future feature page content.

### Demo or Before/After Pics
CleanShot 2023-11-06 at 14 57 38

CleanShot 2023-11-06 at 14 56 51

### Edge Cases
na

### Testing Steps
go to /features/creators 

### Checklist

Please make sure to review and check all of the following:

- [ ] I've tested the changes and all tests pass.
- [ ] (if web) I've tested the changes on various desktop screen sizes to ensure responsiveness.
- [ ] (if mobile) I've tested the changes on both light and dark modes.

### CODE REVIEW

**Comment by jakzaizzat** [comment]
Seems like there is some weird semicolon here 

CleanShot 2023-11-15 at 15 24 46@2x

**Comment by jakzaizzat** [comment]
Not sure is it my monitor or anything, the second text looks blurry. Not blocking this PR, maybe we can try to upload higher resolution image for that 

CleanShot 2023-11-15 at 15 25 53@2x

File: apps/web/src/contexts/globalLayout/GlobalAnnouncementPopover/useGlobalAnnouncementPopover.tsx
Comment (jakzaizzat) [inline apps/web/src/contexts/globalLayout/GlobalAnnouncementPopover/useGlobalAnnouncementPopover.tsx:58]: Seems like the `typecheck` failed because of missing this key. Maybe we need to update the relay schema

**Comment by kaitoo1** [comment]
> Seems like there is some weird semicolon here
> 
> CleanShot 2023-11-15 at 15 24 46@2x

oh nice catch, removed

File: apps/web/src/contexts/globalLayout/GlobalAnnouncementPopover/useGlobalAnnouncementPopover.tsx
Comment (kaitoo1) [reply apps/web/src/contexts/globalLayout/GlobalAnnouncementPopover/useGlobalAnnouncementPopover.tsx:58]: good catch, updated!

**Comment by kaitoo1** [comment]
> Not sure is it my monitor or anything, the second text looks blurry. Not blocking this PR, maybe we can try to upload higher resolution image for that
> 
> CleanShot 2023-11-15 at 15 25 53@2x

just checked and it looks like the content isn't ready yet and we're not going to publish until next week so I'm gonna wait to merge this PR! thanks for reviewing , at least it'll be good to go when we need it 🙏

**Review by Robinnnnn — APPROVED** [review]

**Pull request merged**
